GliaWidgets/View/Call/CallView.swift

```diff
@@ -81,6 +81,54 @@ class CallView: EngagementView {
         adjustLocalVideoFrameAfterLayout()
     }
 
+    override func setup() {
```

Comment:
super.setup() missing here. Is this intentional?

Reply:
It's ok. As described in task "Right now, almost everything is configurable, but the background for the middle section of the chat (the white portion) is not exposed", we need to fix only backgroundColor in chat